name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/PortfolioListView.java

```diff
@@ -256,6 +258,23 @@ public String getText(Object element)
         }));
         portfolioColumns.addColumn(column);
 
+        column = new Column("ref_cash_bal", "Balance of reference account", SWT.RIGHT, 100); //$NON-NLS-1$
```

Comment:
Missing translations

Reply:
I wonder if we should re-use the label from the account view: ColumnBalance

It works nicely in German: Depotvolumen vs. Kontostand

But maybe not so well in other languages.